tests/repl/test_evaluation.ml

```diff
@@ -61,11 +61,11 @@ let test__multiple_phrases ctxt =
   assert_equal ~ctxt ~printer:[%show: status] SHELL_OK status ;
   assert_equal ~ctxt ~printer:[%show: reply list] expected actual
 
-let test__directive ctxt =
+(* let test__directive ctxt =
   let status, actual = eval "#load \"str.cma\" ;; Str.regexp \".*\"" in
```

Comment:
This test case only checks whether a directive works fine, or not. We can check an other directive, not `#load`.
In my environment, this patch works.

```diff
-(* let test__directive ctxt =
-  let status, actual = eval "#load \"str.cma\" ;; Str.regexp \".*\"" in
+let test__directive ctxt =
+  let status, actual = eval "#require \"str\" ;; Str.regexp \".*\"" in
   let expected = [iopub_success ~count:0 "- : Str.regexp = <abstr>\n"] in
```